_1_basics/src/main/java/code/_4_student_effort/FizzBuzz.java

```diff
@@ -0,0 +1,34 @@
+package code._4_student_effort;
+
+public class FizzBuzz {
+    public FizzBuzz() {}; //default constructor
+
+    //method to be tested in Main
+    public void FizzBuzz () {
+        for (int i = 1; i <= 100; i++)
```

Comment:
numarul 105 este divizibil si cu 3,5,7 si ar trebui sa afiseze FizzBuzzRizz 
crezi ca ar exista o varianta mai usoara a.i. sa nu trebuiasca sa adaugi conditii si if-uri pentru fiecare combinatie in parte ?

Reply:
Sa verific daca e divizibil intai de toate cu 3*5*7*11 => sout "FizzBuzzRizzJazz" 
apoi pe rand, divizibil cu 3 => "Fizz"
divizibil cu 5 => "Buzz"
divizibil cu "7" => "Rizz" 
divizibil cu "11" => "Jazz" ? 
Sau e acelasi lucru ?

Reply:
Cu StringBuilder ar fi mai ok?

Reply:
stringBuilder nu o sa te ajute sa elimi toate conditiile ...

te poti uita si prin implementarile altor colegi ... https://github.com/miualinionut/java-training/pulls